tvdb_v4_official.py

```diff
@@ -1,8 +1,5 @@
 import json
 import urllib
-import urllib.parse
-import urllib.request
```

Comment:
When I tested with python 3.8.10 it still requires an "import urllib.request" for the script to work.